ruby/src/security/CWE-770/DoS.ql

```diff
@@ -0,0 +1,150 @@
+/**
+ * @name Denial of Service using unconstrained integer/float value
+ * @description A remote user-controlled integer/float value can reach a condition that controls how many times a repeatable operation can be executed. A malicious user may misuse that value to cause an application-level denial of service.
+ * @kind path-problem
+ * @id rb/dos
```

Comment:
Use a different id (corresponding to the new file name)
The id should be prefixed with `githubsecuritylab/` instead of `rb/`.

Reply:
fixed in the renaming commit https://github.com/GitHubSecurityLab/CodeQL-Community-Packs/pull/55/commits/d56a0bc7f987b55d5a00c7cfdcd68273d12add24